### DISCUSSION TITLE
蓝灯连接上以后打不开任何网站

### DISCUSSION THREAD

**Discussion by jean-reno** [getlantern/lantern-forum-cn#742] [Q&A]
蓝灯连接上以后打不开任何网站，已经很长时间这样了，请官方解决 版本是：7.8.4 (20240402.130121)  一直没有自动更新 自己下载了新版本后，严重的是 根本连不上了

**Comment by MaxMKMax** [comment]
@jean-reno 您好，谢谢您的反馈。请问您用的是哪个设备和系统?

蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by jean-reno** [reply]
windows 10 LTSC
